lib/Segment/Client.php

```diff
@@ -177,8 +177,10 @@ private function message($msg, $def = ""){
     if ($def && !isset($msg[$def])) $msg[$def] = array();
     if ($def && empty($msg[$def])) $msg[$def] = (object)$msg[$def];
 
-    if (!isset($msg["context"])) {
```

Comment:
Don't we still want this check somewhere? I honestly forget how PHP treats it if it's not there but I feel like it should error?